### PR TITLE
Add SP-UUID to event property.

### PR DESCRIPTION
**Pull request by Manchiz**
### Description
Resolve the SP-UUID from the thread-local and include it into event property.

### Related issue
- https://github.com/wso2/product-is/issues/20543

### Todo

- [ ]  bump `carbon-identity-framework` from https://github.com/wso2/carbon-identity-framework/pull/5743

### CODE REVIEW

File: components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java [hunk 1]
```diff
@@ -898,11 +898,16 @@ protected void triggerNotification(String userName, String userStoreDomainName,
             throws AccountLockException {
 
         String eventName = IdentityEventConstants.Event.TRIGGER_NOTIFICATION;
+        String serviceProviderUUID = (String) IdentityUtil.threadLocalProperties.get().get(IdentityEventConstants.EventProperty.SERVICE_PROVIDER_UUID);
 
         HashMap<String, Object> properties = new HashMap<>();
+        if (serviceProviderUUID != null && !serviceProviderUUID.isEmpty()) {
```
Comment (Avarjana) [inline components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java:904]: Can we use isBlank() instead?

File: components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java [hunk 2]
```diff
@@ -898,11 +898,16 @@ protected void triggerNotification(String userName, String userStoreDomainName,
...
+            properties.put(IdentityEventConstants.EventProperty.SERVICE_PROVIDER_UUID, serviceProviderUUID);
+        }
         properties.put(USER_NAME, userName);
         properties.put(IdentityEventConstants.EventProperty.USER_STORE_DOMAIN, userStoreDomainName);
         properties.put(IdentityEventConstants.EventProperty.TENANT_DOMAIN, tenantDomain);
+
```
Comment (Avarjana) [inline components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java:910]: We can get rid of this new line

File: components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java (hunk 1 above)
Comment (Manchiz) [reply components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java]: ack

File: components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java (hunk 2 above)
Comment (Manchiz) [reply components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java:910]: ack

**Review by Avarjana — APPROVED** [review]

**Review by Avarjana — APPROVED** [review]

**Pull request merged**